winmake.ps1

```diff
@@ -362,6 +401,9 @@ switch ($target) {
         Write-Host "Example: Generate the documetation artifacts"
```

Comment:
change "documetation" to "documentation" here too please.

Reply:
Fixed. Thank you.